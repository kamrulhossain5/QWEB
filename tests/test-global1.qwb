int a;
int b;

function none printa(){
  print(a);
}

function none printbb(){
  print(b);
}

function none incab(){
  a = a + 1;
  b = b + 1;
}

function int main(){
  a = 30;
  b = 17;
  printa();
  printbb();
  incab();
  printa();
  printbb();
}
