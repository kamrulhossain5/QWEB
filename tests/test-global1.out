30
17
31
18
